python/CMakeLists.txt

```diff
@@ -263,6 +263,66 @@ function(bundle_boost_lib library_path)
   endif()
 endfunction()
 
+function(bundle_zlib zlib_home)
+  if (MSVC)
```

Comment:
Can FindZLIB.cmake not be used here?

Reply:
We can't use FindZLIB.cmake here.
It finds `zlib.lib` to link zlib as shared library. But we need `zlib.dll` here.

Anyway, I've removed unused codes by https://github.com/apache/arrow/pull/2483/commits/3db0830345d865160d493d10308310b41ee2aa44
The removed codes are for non Windows. I kept them because we may use them later. But we can remove unused codes.

Reply:
OK, no problem. Thank you